src/array_buffer.rs

```diff
@@ -616,3 +623,20 @@ impl ArrayBuffer {
     ))
   }
 }
+
+impl DataView {
+  /// Returns a new DataView.
+  #[inline(always)]
+  pub fn new<'s>(
+    scope: &mut HandleScope<'s>,
+    arraybuffer: Local<'s, ArrayBuffer>,
+    byte_offset: usize,
+    length: usize,
```

Comment:
Would be nice to test `byte_offset` and `length` combinations, but maybe not a big deal